<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeagueDialog</class>
 <widget class="QDialog" name="LeagueDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelTeamName">
       <property name="text">
        <string>Team Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="teamLineEdit">
       <property name="placeholderText">
        <string>Team Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addTeamBtn">
       <property name="text">
        <string>Add a Team</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editTeamBtn">
       <property name="text">
        <string>Edit a Team</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteTeamBtn">
       <property name="text">
        <string>Delete a Team</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importLeagueBtn">
       <property name="text">
        <string>Import League</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportLeagueBtn">
       <property name="text">
        <string>Export League</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="displayTeam"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelLeagueEditor">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Edit League</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LeagueDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LeagueDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
